src/store/settings.ts

```
@@ -77,8 +77,30 @@ export const useSettings = create<AppSettings & SettingsActions>()(
+// 跨窗口同步 Settings 变化
+const settingsBc = new BroadcastChannel("app-settings-sync-channel");
+let isReceivingSettings = false;
+
+settingsBc.onmessage = (ev) => {
+  isReceivingSettings = true;
+  useSettings.setState(ev.data);
+  isReceivingSettings = false;
+};
+
+// 只有在主动修改时进行广播，避免死循环
+useSettings.subscribe((state) => {
```

ESLint

[error] Replace `(state)` with `state`

(prettier/prettier)